src/main/java/org/mariadb/jdbc/internal/stream/PacketOutputStream.java

```diff
@@ -1,617 +1,665 @@
 package org.mariadb.jdbc.internal.stream;
 
-import java.io.*;
-import java.nio.ByteBuffer;
-import java.nio.ByteOrder;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.Reader;
+import java.io.UnsupportedEncodingException;
 import java.sql.Timestamp;
 import java.util.Calendar;
 import java.util.zip.DeflaterOutputStream;
 
+import org.mariadb.jdbc.internal.util.ByteBuffer;
 
 public class PacketOutputStream extends OutputStream {
```

Comment:
Hi Mike,

What do you mean by the "spacing changes" ? the size of the buffer  (1024
-> 4096) ? The target is to have one page size.

Jacques.

On Fri, Mar 4, 2016 at 3:16 AM, Mike Jensen <notifications@github.com>
wrote:

> In src/main/java/org/mariadb/jdbc/internal/stream/PacketOutputStream.java
> <https://github.com/MariaDB/mariadb-connector-j/pull/72#discussion_r54982187>
> :
>
> >  import java.sql.Timestamp;
> >  import java.util.Calendar;
> >  import java.util.zip.DeflaterOutputStream;
> >
> > +import org.mariadb.jdbc.internal.util.ByteBuffer;
> >
> >  public class PacketOutputStream extends OutputStream {
>
> In my opinion these spacing changes make it VERY hard to evaluate this
> commit. If you are going to do spacing changes (though not sure why you
> would in this case), you should probably do it in a separate commit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MariaDB/mariadb-connector-j/pull/72/files#r54982187>.
>